packages/core/spec/screenplay/interactions/Wait.spec.ts

```diff
@@ -106,6 +108,85 @@ describe('Wait', () => {
             })
         });
 
+        it('fails the actor flow when accessing the first item of an empty list after the timeout has expired', async () => {
+
+            const emptyList = List.of([]);
+            const startTime = Date.now();
+
+            await expect(
+                serenity.theActorCalled('Wendy')
+                    .attemptsTo(
+                        Wait.until(emptyList.first(), isGreaterThan(1)),
+                    )
+            ).to.be.rejected.then((error: AssertionError) => {
+                const elapsedTime = Date.now() - startTime;
+                expect(elapsedTime).to.be.greaterThanOrEqual(5000);
+                expect(error.expected).to.be.undefined;
```

Comment:
Not sure why I expect the error to be undefined here. I just copied this from a similar test.

Reply:
Not sure what the original source was, but if I remember correctly, the scenario was throwing an `AssertionError` that didn't have an `expected` and `actual` (that a regular `AssertionError` would have). 
If we're throwing an "element missing" error, making sure that there's no expected and actual is no longer needed since we've introduced a more specialised error.

Reply:
Actually, it it still expected that this test throws an `AssertionError`, so no need to remove it. I think it's right where it is.